olittwhmcs/serializer.py

```diff
@@ -64,6 +64,17 @@ def get_client_request_parameters(email=None, client_id=None):
     return parameters
 
 
+def update_client_request_parameters(**kwargs):
```

Comment:
This function needs a doctstring. You can check some references from [this implementation](https://github.com/truehostcloud/python-whmcs-client/blob/ff183d3044085aaf51c0c9edc83703e851e02a7c/olittwhmcs/serializer.py#L16)